manifests/osfamily/debian.pp

```diff
@@ -7,6 +7,14 @@
     key        => {
       'id'     => '47B320EB4C7C375AA9DAE1A01054B7A24BD6EC30',
       'server' => 'pgp.mit.edu',
-    }
+    },
+    notify     => Notify['pc1_repo_force'],
```

Comment:
The failure at http://jenkins-modules-dev.delivery.puppetlabs.net/job/modules_puppetlabs-puppet_agent_intn-sys_core-master-foss-stable/PLATFORM_CONFIG=ubuntu-1404-64mda,SLAVE_LABEL=beaker/8/console is what I'm fixing here.